grammars/tree-sitter-python.cson

```diff
@@ -73,18 +73,21 @@ scopes:
   'interpolation > "}"': 'punctuation.section.embedded'
 
   'class_definition > identifier': 'entity.name.type.class'
-  'function_definition > identifier': 'entity.name.function'
+  'function_definition > identifier': 'entity.name.function.def'
```

Comment:
this should probably be `entity.name.function.definition` to be more consistent with other languages

Reply:
```suggestion
  'function_definition > identifier': 'entity.name.function.definition'
```

Reply:
Thanks! fixed.